brainstormforce/astra
# AST-2872 - Core Blocks Custom Spacing Mismatch With Frontend

### Description

- The block editor paddings in the customizer are not applicable to the custom layout blocks. The paddings can be seen in the editor but no padding in the front.
- Ideally, it should not apply in editor as well, as we are not yet supporting Custom Layouts Core Blocks Custom Spacing in Frontend.

### Screenshots

- Issue : https://d.pr/v/lwOn17
- Fix : https://d.pr/i/3ezGyS

### Types of changes




- Excluded Block editor Core Block Spacing Support in Editor from Custom Layout Blocks.

### How has this been tested?

- Choose a Custom Layout : either 404, or Header or Footer, or Hooks
- Pick a Group, Column, or Cover block in editor.
- Customizer > Block Editor > Core Blocks Spacing > Custom, apply spacing here.
- It should not apply for Custom Layout Core Blocks such as Group, Column, Cover.
- Check Customizer Preview, it should not apply to Custom Layouts here as well.

### Checklist:
